TAO/tests/Oneway_Send_Timeouts/Server_Task.h

```diff
@@ -3,34 +3,32 @@
 
 #include "Server.h"
 
-#include "ace/Task.h"
 #include "ace/ARGV.h"
-#include "ace/Auto_Ptr.h"
+#include "ace/Task.h"
+
+#include <string>
 
 class Server_Task : public ACE_Task_Base
 {
- public:
-
+public:
   Server_Task (const std::string& args)
     : args_ (args)
-  { }
-  ~Server_Task ()
-    {
-      this->force_shutdown ();
-    }
+  {}
+  ~Server_Task () { this->force_shutdown (); }
 
   virtual int svc ()
   {
     bool initializer = false;
     {
       ACE_GUARD_RETURN (TAO_SYNCH_MUTEX, ace_mon, this->mutex_, -1);
-      if (server_.get() == 0) {
-  ACE_ARGV my_args (args_.c_str());
+      if (server_.get () == 0) {
```

Comment:
No need to call `get ()` when this is now a `unique_ptr`, there is a `operator bool`

Reply:
OK